webapp/src/components/modals/channel_settings/edit_channel_settings.tsx

```diff
@@ -343,15 +343,14 @@ export default class EditChannelSettings extends PureComponent<Props, State> {
                         defaultMessage='Cancel'
                         onClick={this.handleClose}
                     />
-                    {showDeleteButton && (
-                        <FormButton
-                            id='jira-delete-subscription'
-                            type='button'
-                            btnClass='btn-danger pull-left'
-                            defaultMessage='Delete'
-                            onClick={this.handleDeleteChannelSubscription}
-                        />
-                    )}
+                    <FormButton
+                        id='jira-delete-subscription'
+                        type='button'
+                        btnClass='btn-danger pull-left'
+                        defaultMessage='Delete'
+                        disabled={!enableDeleteButton}
+                        onClick={this.deleteChannelSubscription}
```

Comment:
Looks like the `onClick` was reverted back to `deleteChannelSubscription` in the merge here

Reply:
It was.  It was also caught by `make test` which I didn't run before pushing the commit.  Thanks buddy!